### ISSUE TITLE
phpmyadmin errors in php7.2

### ISSUE THREAD

**Issue by dereuromark** [alt3/cakebox#99]
with the now installed PHP 7.2 you get some count() and other errors that prevent you from working with it smoothly.

Can we wget download from https://www.phpmyadmin.net/downloads/ (latest one, e.g. phpMyAdmin 4.7.8) and replace the current 4.5.x one as part of the install script?

**Comment by bravo-kernel** [comment]
phpmyadmin is not part of the built; just an example in the docs

**Comment by dereuromark** [comment]
Yeah but the documentation then needs adjusting at least: http://cakebox.readthedocs.io/en/latest/tutorials/connecting-to-your-databases/
As this will not really work anymore.

**Comment by bravo-kernel** [comment]
I agree, feel free to update the doc with a working instruction (or note).
